ContactsApp/AppServiceLocator.swift

```diff
@@ -1,36 +0,0 @@
-
-//
-//  AppServiceLocator.swift
-//  ContactsApp
-//
-//  Created by Davide Mendolia on 30/09/15.
-//  Copyright © 2015 GoKarumi S.L. All rights reserved.
-//
-
-import Foundation
-
-class AppServiceLocator {
```

Comment:
The idea of this class is to introduce a Service Locator pattern. This is not mention explicitly in the solution, that's the reason I would not like to split this class.